diff --git a/cmake/Modules/FindAWSSDK_EP.cmake b/cmake/Modules/FindAWSSDK_EP.cmake
--- a/cmake/Modules/FindAWSSDK_EP.cmake
+++ b/cmake/Modules/FindAWSSDK_EP.cmake
@@ -38,7 +38,11 @@ if(TILEDB_VCPKG)
   # TODO: We may need to conditionally use ${AWSSDK_PLATFORM_DEPS} here for dynamically linked deps, but
   #       it lists bare "pthread;curl" which leads to linkage of system versions. For static linkage, we
   #       handle those elsewhere at the moment.
+  set(_BUILD_SHARED_LIBS_OLD ${BUILD_SHARED_LIBS})
+  set(BUILD_SHARED_LIBS ON)
   find_package(AWSSDK REQUIRED QUIET COMPONENTS ${AWS_SERVICES})
+  set(BUILD_SHARED_LIBS ${_BUILD_SHARED_LIBS_OLD})
+  unset(_BUILD_SHARED_LIBS_OLD)
 
   if (TILEDB_STATIC)
 
